internal/kibana/status.go

```diff
@@ -0,0 +1,47 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package kibana
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+const SNAPSHOT_SUFFIX = "-SNAPSHOT"
+
+type statusType struct {
+	Version struct {
+		Number        string `json:"number"`
+		BuildSnapshot bool   `json:"build_snapshot"`
+	} `json:"version"`
+}
+
+// Version method returns the version of Kibana (Elastic stack)
+func (c *Client) Version() (string, bool, error) {
```

Comment:
Version method from `internal/kibana/injected_metadata.go` deleted in favor of this one.

This new method uses the `/api/status` endpoint, and it has a dedicated field to indicate whether or not the version is a snapshot one ([build_snapshot](https://github.com/elastic/kibana/blob/main/packages/core/status/core-status-server-internal/src/routes/status.ts#L123)).

Reply:
Sure! Added that new object.